src/main/scala/com/codacy/client/stash/client/StashClient.scala

```diff
@@ -1,21 +1,20 @@
 package com.codacy.client.stash.client
 
-import com.codacy.client.stash.client.auth.OAuthRSACalculator
+import com.codacy.client.stash.client.auth.WSSignatureCalculatorRSA
 import com.codacy.client.stash.util.HTTPStatusCodes
 import com.ning.http.client.AsyncHttpClient
+import com.ning.http.client.oauth.{ConsumerKey, RequestToken}
 import play.api.libs.json.{JsValue, Json, Reads}
-import play.api.libs.oauth.{ConsumerKey, RequestToken}
 import play.api.libs.ws.ning.NingWSClient
-
 import scala.concurrent.Await
 import scala.concurrent.duration._
 
 class StashClient(baseUrl: String, key: String, secretKey: String, token: String, secretToken: String) {
 
-  private lazy val KEY = ConsumerKey(key, secretKey)
-  private lazy val TOKEN = RequestToken(token, secretToken)
+  private lazy val KEY = new ConsumerKey(key, secretKey)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Field name 'KEY' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/Codacy/stash-scala-client/file/2170175716/issues/source?bid=2734487&fileBranchId=3159563#l14)